lib/utils.pm

```diff
@@ -821,6 +821,7 @@ the second run will update the system.
 
 sub ssh_fully_patch_system {
```

Comment:
```suggestion
sub ssh_patch_system_zypper {
```

Reply:
This was the previous original routine name and I did't dare to rename it, so I agree this renaming

Reply:
This name cannot be changed, it was left unchanged because used by other modules too, so adding the above routine dedicated for microos/transact.

Reply:
ok